indico/MaKaC/PDFinterface/conference.py

```diff
@@ -240,55 +231,46 @@ def __init__(self, abstract, tz=None):
 
     @staticmethod
     def _get_status(abstract):
-        status = abstract.getCurrentStatus()
-        #style = ParagraphStyle({})
-        #style.firstLineIndent = -90
-        #style.leftIndent = 90
-        if isinstance(status, review.AbstractStatusDuplicated):
-            original = status.getOriginal()
-            return _("DUPLICATED ({0}: {1})").format(original.getId(), original.getTitle())
-        elif isinstance(status, review.AbstractStatusMerged):
-            target = status.getTargetAbstract()
-            return _("MERGED ({0}: {1})").format(target.getId(), target.getTitle())
+        state_title = abstract.state.title.upper()
+        if abstract.state == AbstractState.duplicate:
+            return _("{} ({}: {})").format(state_title, abstract.duplicate_of.friendly_id, abstract.duplicate_of.title)
+        elif abstract.state == AbstractState.merged:
+            return _("{} ({}: {})").format(state_title, abstract.merged_into.friendly_id, abstract.merged_into.title)
         else:
-            return AbstractStatusList.getInstance().getCaption(status.__class__).upper()
+            return abstract.state.title.upper()
 
     @staticmethod
     def _get_track_judgements(abstract):
         judgements = []
-
-        for track in abstract.getTrackListSorted():
-            status = abstract.getTrackJudgement(track)
-            if isinstance(status, review.AbstractAcceptance):
-                if status.getContribType() is None:
-                    contribType = ""
-                else:
-                    contribType = status.getContribType().name
-                st = _("Proposed to accept: {0}").format(contribType)
-
-            elif isinstance(status, review.AbstractRejection):
+        for track in abstract.reviewed_for_tracks:
+            st = ''
+            track_review_state = abstract.get_track_reviewing_state(track)
+            if track_review_state == AbstractReviewingState.positive:
+                track_reviews = abstract.get_track_reviews(track)
+                # TODO: Handle different contribution type proposals
+                contrib_type = track_reviews[0].proposed_contribution_type
+                contrib_type_title = contrib_type.name
+                if not all(track_review.proposed_contribution_type == contrib_type for track_review in track_reviews):
+                    contrib_type_title = _("Multiple contribution types")
+                st = _("Proposed to accept: {}").format(contrib_type_title)
```

Comment:
Shouldn't we use the same naming as in the UI? That would be using `AbstractReviewingState.positive.title`.